### PR TITLE
Updated to AlmaLinux 9

### PR DESCRIPTION
**Pull request by GrantAOConnor**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ dnf-automatic CHANGELOG
 This file is used to list changes made in each version of the
 dnf-automatic cookbook.
 
+2.0.5 (2023-11-27)
+------------------
+- Add AlmaLinux 9 Support
+
```
Comment (ramereth) [inline CHANGELOG.md]: Please remove this. Our automated system automatically updates this when we merge this.

File: metadata.rb [hunk 2]
```diff
@@ -8,4 +8,4 @@
 description      'Installs/Configures dnf-automatic'
 version          '2.0.4'
 
-supports         'almalinux', '~> 8.0'
+supports         'almalinux', '~> 9.0'
```
Comment (ramereth) [inline metadata.rb]: Please keep 8.0 listed and only add 9.0

File: recipes/default.rb [hunk 3]
```diff
@@ -15,8 +15,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-if node['platform_version'].to_i < 8
-  Chef::Log.warn('dnf-automatic is only supported on EL8 and greater')
+if node['platform_version'].to_i < 9
```
Comment (ramereth) [inline recipes/default.rb]: This needs to stay as this still works on EL8

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -1,9 +1,9 @@
 require 'chefspec'
 require 'chefspec/berkshelf'
 
-ALMA_8 = {
+ALMA_9 = {
   platform: 'almalinux',
-  version: '8',
+  version: '9',
```
Comment (ramereth) [inline spec/spec_helper.rb]: Keep ALMA_8 but add ALMA_9

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Patch bump.

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.5.  Have a nice day!

**Pull request merged**